src/screens/promote/screen/promoteScreen.js

```diff
@@ -0,0 +1,280 @@
+                          </Text>
+                        }
+                      </View>
+
+                      <Autocomplete
+                        autoCapitalize="none"
+                        autoCorrect={false}
+                        inputContainerStyle={styles.autocomplate}
+                        data={permlinkSuggestions}
+                        listContainerStyle={styles.autocomplateListContainer}
+                        listStyle={styles.autocomplateList}
+                        onChangeText={text => this._handleOnPermlinkChange(text)}
+                        renderTextInput={() => (
+                          <TextInput
+                            style={styles.input}
+                            onChangeText={text => this._handleOnPermlinkChange(text)}
+                            value={permlink}
+                            placeholder={intl.formatMessage({ id: 'promote.permlink' })}
+                            placeholderTextColor="#c1c5c7"
+                            autoCapitalize="none"
+                          />
+                        )}
+                        renderItem={({ item, i }) => (
+                          <TouchableOpacity
+                            key={item}
+                            onPress={() =>
+                              this.setState({ permlink: item, permlinkSuggestions: [] })
+                            }
+                          >
+                            <Text style={styles.autocomplateItemText}>{item}</Text>
+                          </TouchableOpacity>
+                        )}
+                      />
+                    </View>
+                  </Fragment>
+
+                  <View style={styles.total}>
+                    <Text style={styles.day}>{`${day} days `}</Text>
+                    <Text style={styles.price}>
+                      {`${get(PROMOTE_PRICING[PROMOTE_DAYS.indexOf(day)], 'price')} eSteem points`}
+                    </Text>
+                  </View>
+
+                  <ScaleSlider
+                    values={[1, 2, 3, 7, 14]}
+                    LRpadding={50}
+                    activeValue={day}
+                    handleOnValueChange={day => this.setState({ day })}
+                    single
+                  />
+                </View>
+                <View style={styles.bottomContent}>
+                  <MainButton
+                    style={styles.button}
+                    isDisable={isLoading}
+                    onPress={() => this.ActionSheet.show()}
+                    isLoading={isLoading}
+                  >
+                    <Text style={styles.buttonText}>
+                      {intl.formatMessage({ id: 'transfer.next' })}
+                    </Text>
+                  </MainButton>
+                </View>
+              </ScrollView>
+            </View>
+            <ActionSheet
+              ref={o => (this.ActionSheet = o)}
```

Comment:
you can use createRef function (you can look contructure method of delegateScreen.js)